pennylane/transforms/insert_ops.py

```diff
@@ -24,13 +24,38 @@
 from pennylane.tape.tape import STATE_PREP_OPS
 from pennylane.transforms.qfunc_transforms import qfunc_transform
 
+# pylint: disable=too-many-branches
+
+
+def _check_position(position):
+    not_op = False
```

Comment:
Add a doc-string here to explain this helper function :
```suggestion
def _check_position(position):
"""Checks the position argument to determine if an operation or list of operations was provided."""
    not_op = False
```